src/main/java/frc/robot/subsystems/StateMachine.java

```diff
@@ -77,7 +79,6 @@ public Command tryState(RobotState desiredState, StateMachine subStateMachine, E
           case STORE_FEEDER:
           case PREP_SPEAKER:
           case PREP_NONE:
-          case PREP_AMP:
```

Comment:
yeahhh switching from prep_shuffle/speaker to prep_amp is light work but switching from prep_amp to prep_shuffle/speaker is harder because we need to get the note back from the drainpipe to the shooter 😅